src/Raven.Studio/typescript/models/database/tasks/ongoingTaskBackupListModel.ts

```diff
@@ -16,8 +16,10 @@ import shell = require("viewmodels/shell");
 class ongoingTaskBackupListModel extends ongoingTaskListModel {
     private static neverBackedUpText = "Never backed up";
     
-    private static namePrefix = "Server Wide Backup";
-
+    private static serverWideNamePrefixFromServer = "Server Wide Backup";
```

Comment:
why do we have this discrepancy?

Reply:
I cannot change this prefix on the server-side - it is too late for that - 
so I can only manipulate the way it looks in the UI